shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -27,7 +30,12 @@
     isInAndroidSdk = false)
 public class ShadowBiometricManager {
 
+  private static final int NO_BIOMETRIC_ERROR = 0;
   protected boolean biometricServiceConnected = true;
+  protected Integer enrolledBiometrics;
```

Comment:
Maybe we don't need to set those fields to `protected`, and keep it `private` if possible.

Reply:
Sure 🆗